modular_zubbers/master_files/code/modules/holiday/holiday.dm

```diff
@@ -0,0 +1,86 @@
+#define PURPLE_LIGHT "#9F79F2"
+#define PURPLE_DARK "#6141A6"
+#define ORANGE_LIGHT "#F27F1B"
+#define ORANGE_DARK "#F24F13"
+#define GREY_LIGHT "#FFFFFF"
+#define GREY_DARK "#292929"
+
+/datum/holiday/halloween
+	name = HALLOWEEN
+	begin_day = 17
+
+/datum/holiday/halloween/New()
+	. = ..()
+	LAZYCLEARLIST(holiday_colors)
+	var/palette = rand(1, 15)
+	switch(palette)
+		if(1, 2)
+			holiday_colors += PURPLE_LIGHT
+			holiday_colors += PURPLE_DARK
+			holiday_colors += GREY_DARK
+			holiday_colors += ORANGE_DARK
+			holiday_colors += ORANGE_LIGHT
+			holiday_colors += GREY_LIGHT
+		if(3)
+			holiday_colors += PURPLE_LIGHT
+			holiday_colors += ORANGE_LIGHT
+			holiday_colors += GREY_LIGHT
+		if(4)
+			holiday_colors += PURPLE_DARK
+			holiday_colors += ORANGE_DARK
+			holiday_colors += GREY_DARK
+		if(5)
+			holiday_colors += pick(GREY_LIGHT, GREY_DARK)
+			holiday_colors += pick(PURPLE_LIGHT, PURPLE_DARK)
+			holiday_colors += pick(ORANGE_LIGHT, ORANGE_DARK)
```

Comment:
Okay so I have no idea if this will work but it should
```suggestion
			holiday_colors += list(
				pick(GREY_LIGHT, GREY_DARK),
				pick(PURPLE_LIGHT, PURPLE_DARK),
				pick(ORANGE_LIGHT, ORANGE_DARK),
				)
```